intelmq/tests/bots/parsers/shadowserver/testdata/compromised_website.csv

```diff
@@ -1,3 +1,3 @@
-"timestamp","ip","port","hostname","tag","application","asn","geo","region","city","url","http_host","category","system","detected_since","server","redirect_target","naics","sic","sector"
+"timestamp","ip","port","hostname","tag","application","asn","geo","region","city","url","http_host","category","system","detected_since","server","redirect_target","naics","sic","sector","cc_url","family"
```

Comment:
are these two columns missing in the data rows?

Reply:
The additional columns are optional.  The CSV spec does not require all fields to be present.